mne/report.py

```diff
@@ -82,7 +82,11 @@ def _scale_mpl_figure(fig, scale):
         sfactor = -((1. / scale) ** 2)
     for text in fig.findobj(mpl.text.Text):
         fs = text.get_fontsize()
-        text.set_fontsize(fs + sfactor)
+        try:
+            text.set_fontsize(fs + sfactor)
```

Comment:
This function needs some checks though. For example, if `scale = 0`, then it will break when evaluating `sfactor` and the user will get an uninformative error.